changelog.d/13627.misc

```diff
@@ -0,0 +1 @@
+Drop unused column `application_services_state.last_txn`.
```

Comment:
As @clokep notes, there's no need for us to increment `SCHEMA_VERSION` and friends, but it might be worth just making a note in the docstring for `SCHEMA_VERSION` (in `synapse/storage/schema/__init__.py`) that this is happening.